CanalTP/CDVNavitiaSDKUX
# New bridge with swift code and new parameters

## How to test:

- Clone this repository
- Create a cordova app
- Add ios and android platform
- Add the plugin referencing the local repository
- Use NavitiaSDKUX.init and NavitiaSDKUX.invokeJourneyResults according to documentation
- Emulate for both iOS and Android
- The journey results screen should appear
